src/docs/Foundations/Typography/HeadingStyles/index.mdx

```diff
@@ -108,6 +108,12 @@ The color of the underline allows you to give visual context to your page or sec
 
 <Preview path="typographyHeading-underlines" nude />
 
+#### Bottom padding
+
+Headings provide with a default bottom padding to keep a right rythm between the other elements of the interface.
```

Comment:
#### Padding bottom 

Headings include a default padding bottom to maintain layout consistency across breakpoints.

or something like that?

Reply:
Here it's a deal about the vertical rhythm inside a page, not about the breakpoints.

Reply:
Headings include a default padding bottom to keep the right vertical rhythm between the interface elements.

then ;-)